README.md

```diff
@@ -177,44 +177,64 @@ Example of a partial CR:
 
 ### Mounting and injecting binding information
 
-This specification allows for data to be mounted using volumes or injected using environment variables.  The best practice is to mount any sensitive information, such as passwords, since that will avoid accidentally exposure via environment dumps and subprocesses.  Also, binding binary data (e.g. .p12 certificate for Kafka) as an environment variable might cause a pod to fail to start (stuck on `CrashLoopBackOff`), so it advisable for backing services with such binding data to mark it with `bindAs: volume`.
+#### Mounting data
```

Comment:
When describing mounting of bindings, I think it's clearer to think about and describe a binding has having a name and two parts, the metadata and secret.  If you think about it this way, the flow of this section of the document talks first about `<mountPathPrefix>/<service-id>` and the two file-per-entry maps underneath it.

Reply:
that's a good approach @nebhale.  One thing I am wondering though, is if `metadata` is a required part, or a (strongly) recommended part.  

A consumer can certainly "get by" with just the `secret`, but with limitations that `metadata` could help unlock (like switching to another compatible binding of the same kind).   

In terms of the spec I was thinking `secret` is a MUST and `metadata` is RECOMMENDED.  What do you think?